dev_process.md

```diff
@@ -0,0 +1,144 @@
+AIRR Standards Development Process
```

Comment:
Rename this file to `CONTRIBUTING.md`. I think GitHub treats it specially in that case also.